src/main/java/org/team1540/bunnybotTank2023/commands/shooter/Shooter.java

```diff
@@ -1,6 +1,64 @@
 package org.team1540.bunnybotTank2023.commands.shooter;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+import org.team1540.bunnybotTank2023.io.shooter.ShooterIO;
+import org.team1540.bunnybotTank2023.io.shooter.ShooterInputsAutoLogged;
+import org.team1540.bunnybotTank2023.utils.AverageFilter;
+import org.team1540.bunnybotTank2023.utils.LoggedTunableNumber;
+
+import static org.team1540.bunnybotTank2023.Constants.*;
 
 public class Shooter extends SubsystemBase {
+
+    private final LoggedTunableNumber kP = new LoggedTunableNumber("Shooter/kP", ShooterConstants.KP);
+    private final LoggedTunableNumber kI = new LoggedTunableNumber("Shooter/kI", ShooterConstants.KI);
+    private final LoggedTunableNumber kD = new LoggedTunableNumber("Shooter/kD", ShooterConstants.KD);
+
+    private final ShooterIO io;
+    private final ShooterInputsAutoLogged inputs = new ShooterInputsAutoLogged();
+
+    private final AverageFilter averageFilter = new AverageFilter(20); // TODO: 11/28/2023 tune filter size
+
+    private double setpoint = 0;
+    private boolean isClosedLoop = false;
```

Comment:
ok but actually why do you need this variable? It's only used in `isAtSetpoint` to return false, which presumably would go unused when you're not trying to do closed loop control